pkg/commands/config.go

```diff
@@ -58,14 +58,19 @@ func getBaseImage(bo *options.BuildOptions) build.GetBase {
 
 		// Using --platform=all will use an image index for the base,
 		// otherwise we'll resolve it to the appropriate platform.
-		//
+		allPlatforms := len(bo.Platforms) == 1 && bo.Platforms[0] == "all"
+
 		// Platforms can be comma-separated if we only want a subset of the base
 		// image.
```

Comment:
It'd be great to update this comment to reflect the new behavior.

Reply:
ditto — comma-separated is still supported from CLI